react/src/theme/components/ProgressIndicator.ts

```diff
@@ -0,0 +1,40 @@
+import { createMultiStyleConfigHelpers } from '@chakra-ui/react'
+import { anatomy } from '@chakra-ui/theme-tools'
+
+const parts = anatomy('attachment').parts(
+  'circleIndicator',
+  'activeIndicator',
+  'container',
+)
+
+const { definePartsStyle, defineMultiStyleConfig } =
+  createMultiStyleConfigHelpers(parts.keys)
+
+const baseStyle = definePartsStyle(({ isActiveIndicator, colorMode }) => ({
+  circleIndicator: {
+    width: '0.75rem',
+    height: '0.75rem',
+    padding: '0.125rem',
+    borderRadius: 'full',
+    backgroundColor: 'interaction.support.unselected',
+    mr: isActiveIndicator ? '1.25rem' : '0.25rem',
+    backgroundClip: 'content-box',
+  },
+  activeIndicator: {
+    // Top required to align it with CircleIndicators
+    top: '0.125rem',
+    width: '1.5rem',
+    height: '0.5rem',
+    borderRadius: 'full',
+    backgroundColor:
+      colorMode === 'light'
```

Comment:
Seems overkill for this. Could we just use the `_dark` property to denote the color in dark mode? Or could we just add a color prop since that is the only thing light or dark mode controls?

If we want dark/light mode differentiation:
```ts
activeIndicator: {
  ...
  bg: 'interaction.support.selected',
  _dark: {
    bg: 'base.content.inverse'
  }
}
```

else maybe just have a default color and allow users of the component to override it with the `bg` prop. (I'm leaning to this)

Also also prefer `bg` to `backgroundColor` prop for consistency.

Reply:
done! preferred the dark/light as per hte suggestion (never knew about `_dark` lmao) cos i think the default dark mode colour will be good enough most of the time. it should still allow customisation iirc